test/functional/interface_zmq.py

```diff
@@ -118,8 +120,10 @@ def _zmq_test(self):
         self.log.info("Test the getzmqnotifications RPC")
         assert_equal(self.nodes[0].getzmqnotifications(), [
             {"type": "pubhashblock", "address": ADDRESS, "hwm": 1000},
+            {"type": "pubhashinstantsenddoublespend", "address": ADDRESS, "hwm": 1000},
             {"type": "pubhashtx", "address": ADDRESS, "hwm": 1000},
             {"type": "pubrawblock", "address": ADDRESS, "hwm": 1000},
+            {"type": "pubrawinstantsenddoublespend", "address": ADDRESS, "hwm": 1000},
```

Comment:
pls add corresponding parts to `test/functional/interface_zmq_dash.py` instead

Reply:
Thanks for pointing to `test/functional/interface_zmq_dash.py`, that's indeed the right place for Dash-specific messages.